tests/modules/test_time.py

```diff
@@ -34,3 +31,147 @@ def test_sleep_negative(self):
                 print(err)
             print('Done.')
             """)
+
+    @unittest.expectedFailure
+    def test_struct_time_valid(self):
+        """
+        valid construction
+        """
+
+        # TODO: this won't pass until the follownig types have support __getitem__()
```

Comment:
In this case, we should either break this test into two parts so that we can mark the failing types, but confirm the working ones actually work; or we should just implement `__getitem__`.